### DISCUSSION TITLE
The tracker function does not work it shows an error as module "'numpy' has no attribute 'float'."

### DISCUSSION THREAD

**Discussion by AarizZafar** [roboflow/notebooks#253] [Q&A]
def detections2boxes(detections: Detections) -> np.ndarray:
    return np.hstack((detections.xyxy, detections.confidence[:, np.newaxis]))
   
 tracks = byte_tracker.update(
        output_results = detections2boxes(detections),
        img_info=frame.shape,
        img_size=frame.shape
      )

![image](https://github.com/roboflow/notebooks/assets/95307132/2b063113-225e-47b3-9460-c205cb683a0a)

      
      The error ->
      
      AttributeError: module 'numpy' has no attribute 'float'.
`np.float` was a deprecated alias for the builtin `float`. To avoid this error in existing code, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations

**Comment by hamm0o0d** [comment]
I have the same problem and I tried to use float instead of np.float but the same error still exists
